.gitlab-ci.yml

```diff
@@ -86,3 +102,23 @@ publish:s3:
       done
   only:
     - master
+
+publish:s3:latest:
+  only:
+    variables:
+      - $MENDER_VERSION =~ /[0-9]+\.[0-9]+\.[0-9]+/
+  stage: publish
+  image: debian:buster
+  dependencies:
+    - build
+  before_script:
+    - apt update && apt install -yyq awscli
+  script:
+    - echo "Publishing latest packages to S3"
+    - deb_version=$(cat output/mender-client-deb-version)
```

Comment:
This is the variable that we need to override to be "latest".

As discussed in person, this will just make a copy of the latest tagged mender client deb package in S3.